infra/airflow-cluster/docker-compose.yaml

```diff
@@ -93,6 +93,7 @@ x-airflow-common:
   extra_hosts:
     - "hermes:65.108.123.44"
     - "newton:65.109.3.225"
+    - "platon:65.108.254.98"
```

Comment:
Can we avoid using IP's here? 
We could tie the machines to some domains. Let me know if this works for you.